website_sale_hide_price/views/website_sale_template.xml

```diff
@@ -47,10 +47,7 @@
         </xpath>
     </template>
     <template id="product_quantity" inherit_id="website_sale.product_quantity">
-        <xpath
-            expr="//div[contains(@t-attf-class, 'css_quantity')]"
-            position="attributes"
-        >
```

Comment:
Can someone explain this change? Currently, it looks like this is breaking the website_sale_hide_price module. I can't tell if this was intentional (and if so why it's on a completely unrelated PR) or a mistake?

Reply:
It seems unintentional. @pilarvargas-tecnativa can you please check?

Reply:
I've raised a PR to revert it at least for 17.0.